datalake/src/main/resources/duties/7.2.7/aws/medium_duty_ha.json

```diff
@@ -54,8 +54,8 @@
         "instanceType": "t3.medium",
         "attachedVolumes": [
           {
-            "count": 1,
-            "size": 100,
+            "count": 0,
+            "size": 0,
```

Comment:
If we set "count" to 0, do we also have to set "size" to 0. That's a little confusing. We should remove the "size" field from this JSON if count is 0.